config.json

```diff
@@ -27,9 +27,13 @@
   "dataPath": "https://regensburg.freifunk.net/data/",
   "reverseGeocodingApi": "https://nominatim.openstreetmap.org/reverse",
   "siteName": "Freifunk Regensburg",
-  "showContact": false,
   "maxAge": 14,
   "nodeZoom": 18,
+  "nodeInfobox":{
+    "showContact": true,
+    "hideMemory": false,
```

Comment:
Should we split setting bout current hardware usage? Both are more or less bind together.